cpp/memilio/mobility/metapopulation_mobility_instant.h

```diff
@@ -302,8 +384,44 @@ class MigrationEdge
     bool m_return_migrated;
     double m_t_last_dynamic_npi_check               = -std::numeric_limits<double>::infinity();
     std::pair<double, SimulationTime> m_dynamic_npi = {-std::numeric_limits<double>::max(), SimulationTime(0)};
+    std::vector<std::vector<size_t>> m_save_indices; // groups of indices from compartments to save
+    TimeSeries<double> m_mobility_results; // save results from edges + entry for the total number of commuters
+
+    /**
+     * Computes a condensed version of m_migrated and puts it in m_mobility_results.
+     * m_mobility_results then only contains commuters with infection states InfectedNoSymptoms and InfectedSymptoms.
+     * Additionally, the total number of commuters is stored in the last entry of m_mobility_results.
+     * @param[in] t current time
+     */
+    void condense_m_mobility(const double t);
```

Comment:
```suggestion
    void add_mobility_result_time_point(const double t);
```
Or something similar?